hail/python/hail/methods/statgen.py

```diff
@@ -2925,7 +2926,7 @@ def filter_alleles(mt: MatrixTable,
     If all alternate alleles of a variant are filtered out, the variant itself
     is filtered out.
 
-    **Using _f_**
+    **Using** `f`
```

Comment:
Just above this, there is another fix needed I think

old_to_new and new_to_old are inconsistent with old_locus and old_alleles. Similarly above `filter_alleles_ht`

![Screenshot 2019-11-04 14 47 22](https://user-images.githubusercontent.com/5543229/68152683-2367a500-ff12-11e9-9e43-ebbaf6ac761a.png)

Reply:
Yup, good catch.

Reply:
oof, good eye.